doc/FAQ

```diff
@@ -0,0 +1,101 @@
+- NodeOS - Lightweight operating system using Node.js as userspace - FAQ -
+--------------------------------------------------------------------------
+
+
+1. NodeOS is a fantastic project, what can I do to help it?
+
+- Linux kernel programmers
+- Node.js programmers
+- Designers
+- Testers
```

Comment:
I guess it would be cool to point out in what areas someone could apply to, but I didn't really know how to discribe it and less yet where to point for specific threads or documentation to look out to get started.